FhirToDataLake/Microsoft.Health.Fhir.Synapse.sln

```diff
@@ -34,14 +30,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synap
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synapse.DataWriter.UnitTests", "test\Microsoft.Health.Fhir.Synapse.DataWriter.UnitTests\Microsoft.Health.Fhir.Synapse.DataWriter.UnitTests.csproj", "{5C62D31B-BE20-4A86-9D04-ABC74F387C71}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synapse.Parquet.CLR.UnitTests", "test\Microsoft.Health.Fhir.Synapse.Parquet.CLR.UnitTests\Microsoft.Health.Fhir.Synapse.Parquet.CLR.UnitTests.csproj", "{67C35B59-5D0C-4846-BE4C-08BF580229E9}"
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Microsoft.Health.Fhir.Synapse.Parquet.Native.UnitTests", "test\Microsoft.Health.Fhir.Synapse.Parquet.Native.UnitTests\Microsoft.Health.Fhir.Synapse.Parquet.Native.UnitTests.vcxproj", "{9AA63275-45BE-4019-B5F1-E371A985D045}"
-EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synapse.Core.UnitTests", "test\Microsoft.Health.Fhir.Synapse.Core.UnitTests\Microsoft.Health.Fhir.Synapse.Core.UnitTests.csproj", "{8805CF56-8937-4FF9-8CF0-3450258C9DE6}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synapse.E2ETests", "test\Microsoft.Health.Fhir.Synapse.E2ETests\Microsoft.Health.Fhir.Synapse.E2ETests.csproj", "{01C19DA0-63E6-43E2-B421-12CC8BD7EB94}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.Synapse.Parquet.UnitTests", "test\Microsoft.Health.Fhir.Synapse.Parquet.UnitTests\Microsoft.Health.Fhir.Synapse.Parquet.UnitTests.csproj", "{35755AAC-C867-499E-8F49-A141ADDD54FA}"
```

Comment:
Is it properly to add `Parquet.unitTests` here?

Reply:
Removed. Thanks